frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -186,7 +188,9 @@ function ReaderBookmark:onShowBookmark()
         if not self.ui.document.info.has_pages then
             page = self.ui.document:getPageFromXPointer(page)
         end
-        v.text = _("Page") .. " " .. page .. " " .. v.notes .. " @ " .. v.datetime
+        if v.text == nil or v.text == "" then
+            v.text = _("Page") .. " " .. page .. " " .. v.notes .. " @ " .. v.datetime
```

Comment:
This is from before I added the template function. Should we turn it into `Page %1 %2 @ %3` even though that looks a little weird?

Reply:
Of course :)